js/app.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+// module for the overall app --> includes here dependencies, routes, controllers, etc
+
+(function(){
+  angular
+  .module('radioApp', [
+    "ui.router",
+    "songs",
+  ])
+  .config(['$stateProvider', RouteBuilder]);
+
+  function RouteBuilder($stateProvider){
+    $stateProvider
+    .state('welcome', {
+      url: '/welcome',
+      templateUrl: "welcome.html",
+    })
+    .state("songsIndex", {
+      url: "/songs",
+      templateUrl:"js/songs/index.html",
+      controller: "songsIndexController",
+      controllerAs: "songsIndexVM"
+    })
+    .state("songsShow", {
+      url: "/songs/:id",
+      templateUrl: "js/songs/show.html",
+      controller: "songsShowController",
+      controllerAs: "songsVM"
```

Comment:
Great job here defining your `RouteBuilder` function, I would probably name your `songsShow` controller as to `songsShowVM` instead just to keep it consistent with the `songsIndexVM`